### DISCUSSION TITLE
Welcome to API Discussions!

### DISCUSSION THREAD

**Discussion by Mafia-7777** [MonkeDev/API#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mafia-7777** [comment]
Ayo whats this 👀

**Reply by Assassin-1234** [reply]
sex

**Comment by birbified** [comment]
pp small vro

**Comment by birbified** [comment]
become birb and get big pp

**Reply by BurnedBed** [reply]
my face when bruh

**Comment by birbified** [comment]
also birb best thx for birb stuff

**Comment by birbified** [comment]
oops forgot to tell about meself

**Comment by birbified** [comment]
i am a birb

**Reply by Mafia-7777** [reply]
Cool, so you're a furry?

**Reply by Rachniotov** [reply]
Guess so

**Reply by BurnedBed** [reply]
no

**Comment by RylanMcMullan** [comment]
where banana

**Reply by BurnedBed** [reply]
no

**Comment by Rachniotov** [comment]
AYO

**Reply by BurnedBed** [reply]
ok

**Comment by JaxGamez** [comment]
is da bruh moment?

**Reply by BurnedBed** [reply]
not bruh!
